src/pages/app/projects/index.tsx

```diff
@@ -25,7 +25,7 @@ export default function ProjectsRoute({
   useEffect(() => {
     if (
       !projectType ||
-      Object.values(ProjectType).every((t) => t !== projectType)
+      Object.values(projectType).every((t) => t !== projectType)
```

Comment:
this should be EProjectType... this checks whether `projectType` is query param is one of know `EProjectType`

Reply:
fixed